packages/opentelemetry-remote-sampler-aws-xray/test/AWSXRayRemoteSampler.test.ts

```diff
@@ -0,0 +1,153 @@
+                    "ServiceType": "AWS::Foo::Bar",
+                    "URLPath": "*",
+                    "Version": 1
+                }
+            },
+            {
+                "CreatedAt": 0.0,
+                "ModifiedAt": 1.611564245E9,
+                "SamplingRule": {
+                    "Attributes": {},
+                    "FixedRate": 0.05,
+                    "HTTPMethod": "*",
+                    "Host": "*",
+                    "Priority": 10000,
+                    "ReservoirSize": 1,
+                    "ResourceARN": "*",
+                    "RuleARN": "arn:aws:xray:us-west-2:123456789000:sampling-rule/Default",
+                    "RuleName": "Default",
+                    "ServiceName": "*",
+                    "ServiceType": "*",
+                    "URLPath": "*",
+                    "Version": 1
+                }
+            },
+            {
+                "CreatedAt": 1.676038494E9,
+                "ModifiedAt": 1.676038494E9,
+                "SamplingRule": {
+                    "Attributes": {},
+                    "FixedRate": 0.2,
+                    "HTTPMethod": "GET",
+                    "Host": "*",
+                    "Priority": 1,
+                    "ReservoirSize": 10,
+                    "ResourceARN": "*",
+                    "RuleARN": "arn:aws:xray:us-west-2:123456789000:sampling-rule/Rule2",
+                    "RuleName": "Rule2",
+                    "ServiceName": "FooBar",
+                    "ServiceType": "*",
+                    "URLPath": "/foo/bar",
+                    "Version": 1
+                }
+            }
+        ]
+    };
+
+    let clock: sinon.SinonFakeTimers;
+    let sampler: AWSXRayRemoteSampler;
+    let axiosPostSpy: sinon.SinonSpy;
+
+    before(() => {
+        nock('http://127.0.0.1:2000')
+            .persist()
+            .post('/GetSamplingRules')
+            .reply(200, getSamplingRulesResponseStub);
+
+
+    });
+
+    beforeEach(() => {
+        clock = sinon.useFakeTimers();
+        axiosPostSpy = sinon.spy(axios, 'post');
+        sampler = new AWSXRayRemoteSampler('http://127.0.0.1:2000', 60 * 1000);
+    });
+
+    afterEach(() => {
+        clock.restore();
+        axiosPostSpy.restore();
+    });
+
+    it('should throw TypeError when an invalid polling interval is passed in', async () => {
+        assert.throws(() => new AWSXRayRemoteSampler('http://127.0.0.1:2000', 0), TypeError);
+        assert.throws(() => new AWSXRayRemoteSampler('http://127.0.0.1:2000', 1.5), TypeError);
+
+    });
+
+
+    it('should make a POST request to the /GetSamplingRules endpoint', async () => {
+
+        clock.tick(60 * 1000);
+        sinon.assert.calledOnce(axiosPostSpy);
+
+    });
+
+
+    it('should make 3 POST requests to the /GetSamplingRules endpoint after 3 intervals have passed', async () => {
+
+        clock.tick(60 * 1000);
+        clock.tick(60 * 1000);
+        clock.tick(60 * 1000);
```

Comment:
nit: You are using `60 * 1000` multiple times in the tests, but the connection is not trivial to figure out. If you extract this value into a variable like `const pollingInterval = 60 * 1000` then it's clear that the clock jumps are for one interval and this is the same interval supplied to `new AWSXRayRemoteSampler` above